### DISCUSSION TITLE
auto numbering in issues

### DISCUSSION THREAD

**Discussion by calbaker** [community/community#69240] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Issues

### Body

When composing an issue with a numbered list, when I hit enter, a new numbered item is generated with the number automatically advanced, as in
```
1. Steal underpants
2. ...
```
which renders as:
1. Steal underpants
2. ...

but since Markdown supports automatic numbering upon rendering, I'd rather just have it do this:
```
1. Steal underpants
1. ...
```
which also renders as:
1. Steal underpants
1. ...

This way, if I realize that I need to change the order of things, I can just move things around without having to also edit a number.  

I think this also applies to Wiki editing.  

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by nreinicke** [comment]
I agree, this feature would be great to have!
